hw6_epam_traning/src/main/java/com/mapLine/MapCreator.java

```diff
@@ -0,0 +1,30 @@
+package com.mapLine;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class MapCreator {
+
+    public MapCreator() {
+    }
+
+    public static Map<Character, Integer> mapCreator(String line) {
+        if (line == null) {
+            throw new LineIsNull("Вы не ввели строку");
+        } else {
+            String value = line.replaceAll("\\s+", "");
+            char[] wordsList = value.toCharArray();
+            Map<Character, Integer> wordsMap = new TreeMap(Collections.reverseOrder());
+            char[] var5 = wordsList;
```

Comment:
var5, var6, var7

![what-the-hell-is-going-on-here](https://user-images.githubusercontent.com/30556705/138248784-1875f725-0ba3-4c11-9194-84ce62402592.jpeg)
?